CHANGELOG-1.10.md

```diff
@@ -687,6 +687,8 @@ If no featureGates was specified in `kubeadm-config`, just change `featureGates:
 
 * Due to an issue in ingress-gce controller in 1.10.0, creating multi-cluster ingresses with [kubemci CLI](https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress) fails. This will be fixed in 1.10.1 release. ([GoogleCloudPlatform/k8s-multicluster-ingress#175](https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress/issues/175))
 
+* Pods on different nodes mount Ceph RBD PVC stuck on ContainerCreating.When set accessModes: ReadOnlyMany, and create pods using the claim, only one pod would be running and the others on different nodes would get stuck on ContainerCreating. But if the pods are all on same node, it works fine.This will be fixed in ([#65124](https://github.com/kubernetes/kubernetes/pull/65124))
```

Comment:
Why do we need to add this to the known issues if it's going to be fixed by this PR?

Reply:
caution the person who will be upgrading 1.10 and record know issues.

Reply:
If changelog is in dispute, you can merge fix bug code first.

Reply:
I don't think we need to add this to known issues because it will be fixed in this release.  However, you should add a Release Note to the initial comment of this PR saying that this issue is fixed.  When the next 1.10 patch goes out, release notes will automatically be generated.